src/lib/blocks.js

```diff
@@ -32,23 +32,100 @@ module.exports = function (vm) {
     const backdropsMenu = function () {
         return vm.runtime.targets[0].sprite.costumes.map(costume => [costume.name, costume.name]);
     };
+    
+    const spriteMenu = function () {
+        var i = 0;
+        var sprites = [];
+        var k = 0;
+        for (i = 0; i < vm.runtime.targets.length; i++) {
```

Comment:
Please use `for (var targetId in vm.runtime.targets)` since then we won't need these iterator variables.